tests/main/snap-interface/task.yaml

```diff
@@ -0,0 +1,11 @@
+summary: Check that "snap interface" works as expected
+details: |
+    The "snap interface" command displays a listing of used interfaces
+execute: |
+    snap interface | MATCH 'core-support\s+ special permissions for the core snap'
+    snap interface --all | MATCH 'classic-support\s+ special permissions for the classic snap'
+    snap interface core-support > out.yaml
+    diff -u out.yaml snap-interface-core-support.yaml
```

Comment:
When I was reading this I was wondering where snap-interface-core-support.yaml comes from, maybe just create it in the task.yaml itself via a here shell construct?